content/manuals/build/ci/github-actions/multi-platform.md

```diff
@@ -284,8 +282,7 @@ target "image-all" {
   inherits = ["image"]
   platforms = [
     "linux/amd64",
-    "linux/arm/v6",
-    "linux/arm/v7",
```

Comment:
I think we could keep the platforms as they can build natively on arm64 runners. So maybe change:

```
runs-on: ${{ matrix.platform == 'linux/arm64' && 'ubuntu-24.04-arm' || 'ubuntu-latest' }}
```

to

```
runs-on: ${{ startsWith(matrix.platform, 'linux/arm') && 'ubuntu-24.04-arm' || 'ubuntu-latest' }}
```

Reply:
Good one